.gitignore

```diff
@@ -1,11 +1,8 @@
 # Cargo stuff
-/target
+**/target
```

Comment:
This change was created because if you built any of the subordinate crates, such as gen_reflection, it would be included by Git.